test/integration/acceptance/set_account_detail_test.cpp

```diff
@@ -147,3 +167,69 @@ TEST_F(SetAccountDetail, WithGrantablePerm) {
         ASSERT_EQ(block->transactions().size(), 1);
       });
 }
+
+/**
+ * C276
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with max key
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, HugeKey) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms())
+      .skipProposal()
+      .skipBlock()
+      .sendTxAwait(
+          complete(baseTx(kUserId, std::string(64, 'a'), kValue)),
+          [](auto &block) { ASSERT_EQ(block->transactions().size(), 1); });
+}
+
+/**
+ * C277
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with empty key
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, EmptyKey) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms())
+      .skipProposal()
+      .skipBlock()
+      .sendTx(complete(baseTx(kUserId, "", kValue)), checkStatelessInvalid);
+}
+
+/**
+ * C278
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with empty value
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, EmptyValue) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms())
+      .skipProposal()
+      .skipBlock()
+      .sendTxAwait(complete(baseTx(kUserId, kKey, "")), [](auto &block) {
```

Comment:
Please move exceptional value as a const var to the beginning of the test case.
The reason is the same as in the first review comment.